src/gui/sdlmain.cpp

```diff
@@ -1040,9 +1024,10 @@ static void setup_presentation_mode(FrameMode &previous_mode)
 	VGA_SetHostRate(host_rate);
 	const auto dos_rate = VGA_GetPreferredRate();
 
-	// Update the VFR duplicate countdown based on the DOS rate to produce a
-	// fixed lower-bound dupe refresh rate.
-	set_vfr_dupe_countdown_from_rate(dos_rate);
+	// Calculate the maximum number of skippable frames before presenting
+	const auto max_skippable = std::ceil(dos_rate / sdl.frame.vfr_min_rate);
+	sdl.frame.vfr_max_skippable = check_cast<int16_t>(iround(max_skippable));
+	assert(sdl.frame.vfr_max_skippable > 0);
```

Comment:
Hmm, can't we just maintain a fractional max skippable value and check against that? Is there something to gain here by rounding up?

Reply:
With a fixed min rate, this rounding will go away :slightly_smiling_face: